src/app/query/bitcoin/address/utxos-by-address.hooks.ts

```diff
@@ -22,12 +22,56 @@ export function filterUtxosWithInscriptions(
 }
 
 /**
- * Warning: ⚠️ These are **all** UTXOs, including Stamped and Inscribed UTXOs.
- * You should probably use `useCurrentNativeSegwitAccountSpendableUtxos` instead.
+ * Warning: ⚠️ To avoid spending inscriptions, when using UTXOs
+ * we set `filterInscriptionUtxos` and `filterPendingTxsUtxos` to true
  */
-export function useCurrentNativeSegwitUtxos() {
+export function useCurrentNativeSegwitUtxos(
+  { filterInscriptionUtxos, filterPendingTxsUtxos } = {
+    filterInscriptionUtxos: true,
+    filterPendingTxsUtxos: true,
+  }
+) {
```

Comment:
```suggestion
interface useCurrentNativeSegwitUtxos {
  filterInscriptionUtxos: boolean;
  filterPendingTxsUtxos: boolean;
}
export function useCurrentNativeSegwitUtxos({
  filterInscriptionUtxos = true,
  filterPendingTxsUtxos = true,
}: UseCurrentNativeSegwitUtxosOptions) {
```

Reply:
you'll need to pass empty object then `useCurrentNativeSegwitUtxos({})`

Reply:
you'll need to pass empty object then `useCurrentNativeSegwitUtxos({})`

Reply:
Hmm, but if you pass just one then the other is `undefined`?

What about

```ts
const defaultArgs: UseCurrentNativeSegwitUtxosArgs = {
  filterInscriptionUtxos: true,
	filterPendingTxsUtxos: true,
};
interface UseCurrentNativeSegwitUtxosArgs {
	  filterInscriptionUtxos?: boolean;
	  filterPendingTxsUtxos?: boolean;
	}
export function useCurrentNativeSegwitUtxos(args: UseCurrentNativeSegwitUtxosOptions) {
  const { filterPendingTxsUtxos, filterInscriptionUtxos } = { ...defaultArgs, ...args };
```

Reply:
may be just 
```const defaultArgs = {
  filterInscriptionUtxos: true,
  filterPendingTxsUtxos: true,
};
export function useCurrentNativeSegwitUtxos(args = defaultArgs) {
  const { filterInscriptionUtxos, filterPendingTxsUtxos } = args;```

Reply:
> Hmm, but if you pass just one then the other is undefined?
yes, why not?